src/scss/_tooltips.scss

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+[data-tooltip] {
+  position: relative;
+
+  cursor: pointer;
+}
+
+[data-tooltip]:before,
+[data-tooltip]:after {
+  position: absolute;
+
+  -webkit-transform: translate3d(0, 0, 0);
+  -moz-transform: translate3d(0, 0, 0);
+  transform: translate3d(0, 0, 0);
+  visibility: hidden;
+  opacity: 0;
+
+  pointer-events: none;
+}
+
+[data-tooltip]:before {
+  z-index: 9999;
+
+  background: transparent;
+  border: 0.1rem solid transparent;
+
+  content: '';
+}
+
+[data-tooltip]:after {
+  z-index: 9998;
+
+  margin: 0.75rem 0;
+  padding: 0.2rem 0.4rem;
+
+  font-size: 0.8rem;
+  line-height: 1.2;
+
+  content: attr(data-tooltip);
+}
+
+[data-tooltip]:hover:before,
```

Comment:
while this is not wrong, let's use the proper `::` for pseudo-selectors. it'll also be consistent with other instances.

Reply:
to clarify:
`[data-tooltip]:hover:before` -> `[data-tooltip]:hover::before`

Reply:
got it..... another great catch. Done! 🦖